### PR TITLE
Specific bridge staff radio

### PR DESCRIPTION
**Pull request by Corsaka**
## About The Pull Request
Introduces a specific bridge staff radio and renames the HoP radio to XO radio

## Why It's Good For The Game
HoP to XO rename is nice, giving bridge staff less theoretical power is nicer

I've been hearing lots of people wondering why bridge staff has the HoP radio, so this fixes that.

## Changelog
:cl:
add: Bridge staff radio
tweak: HoP radio renamed to XO radio
/:cl:

### CODE REVIEW

File: nsv13/code/game/objects/items/munitions_items.dm [hunk 1]
```diff
@@ -40,6 +40,11 @@
 	channels = list(RADIO_CHANNEL_MUNITIONS = 1, RADIO_CHANNEL_SUPPLY = 1)
 	independent = TRUE
 
+/obj/item/encryptionkey/headset_bridge
+	name = "bridge radio encryption key"
+	icon_state = "com_cypherkey"
+	channels = list(RADIO_CHANNEL_SUPPLY = 1, RADIO_CHANNEL_SERVICE = 1, RADIO_CHANNEL_COMMAND = 1)
```
Comment (francinum) [inline nsv13/code/game/objects/items/munitions_items.dm:46]: This is completely identical to the XO key as it stands. Does this really justify a new type to begin with?

File: code/game/objects/items/devices/radio/encryptionkey.dm [hunk 2]
```diff
@@ -105,9 +105,9 @@
 	icon_state = "cmo_cypherkey"
 	channels = list(RADIO_CHANNEL_MEDICAL = 1, RADIO_CHANNEL_COMMAND = 1)
 
-/obj/item/encryptionkey/heads/hop
```
Comment (francinum) [inline code/game/objects/items/devices/radio/encryptionkey.dm:108]: Tag as NSV Change.

File: code/game/objects/items/devices/radio/headset.dm [hunk 3]
```diff
@@ -220,11 +220,11 @@ GLOBAL_LIST_INIT(channel_tokens, list(
 	icon_state = "com_headset"
 	keyslot = new /obj/item/encryptionkey/heads/cmo
 
-/obj/item/radio/headset/heads/hop
-	name = "\proper the head of personnel's headset"
+/obj/item/radio/headset/heads/xo
```
Comment (francinum) [inline code/game/objects/items/devices/radio/headset.dm:223]: Tag as NSV Change.

File: nsv13/code/game/objects/items/munitions_items.dm (hunk 1 above)
Comment (Karmic-Skink) [reply nsv13/code/game/objects/items/munitions_items.dm:46]: It doesn't need a new type and this is not a munitions item so would not be in this file.

File: nsv13/code/game/objects/items/munitions_items.dm [hunk 4]
```diff
@@ -70,6 +75,12 @@
 	icon_state = "sec_headset"
 	keyslot = new /obj/item/encryptionkey/munitions_tech
 
+/obj/item/radio/headset/headset_bridge
```
Comment (Karmic-Skink) [inline nsv13/code/game/objects/items/munitions_items.dm:78]: Not a munitions item.

File: nsv13/code/game/objects/items/munitions_items.dm (hunk 1 above)
Comment (Corsaka) [reply nsv13/code/game/objects/items/munitions_items.dm:46]: The idea was that the bridge officer isn't given the HoP encryption key if they ever needed a new one, but yeah, it's probably useless.

File: nsv13/code/game/objects/items/munitions_items.dm (hunk 4 above)
Comment (Corsaka) [reply nsv13/code/game/objects/items/munitions_items.dm:78]: Bridge Staff is under the munitions header in the Job Select screen, hence...

Comment (Karmic-Skink) [reply nsv13/code/game/objects/items/munitions_items.dm:73]: It is under the Job Select screen to avoid certain issues. Hence not.

Comment (Corsaka) [reply nsv13/code/game/objects/items/munitions_items.dm:73]: So am I to place this in a separate file and add it to the #includes?

Comment (Karmic-Skink) [reply nsv13/code/game/objects/items/munitions_items.dm:73]: Yes a separate file and add it to the .dme

**Pull request merged**